src/Frontend/Assets.php

```diff
@@ -42,6 +44,11 @@ class Assets
      */
     protected $assetsDir;
 
+    /**
+     * @var SettingsRepositoryInterface
+     */
+    protected $settings;
```

Comment:
Were you planning on adding this to the constructor?

Reply:
Yes, but I ran into some issues I didn't understand, so swapped to resolve. If you can get it working... 🤞